dashboard/config/routes.rb

```diff
@@ -153,6 +153,7 @@ def redirect_to_teacher_dashboard
   post '/milestone/:user_id/level/:level_id', :to => 'activities#milestone', :as => 'milestone_level'
   post '/milestone/:user_id/:script_level_id', :to => 'activities#milestone', :as => 'milestone'
 
+  get '/admin/monthly', to: 'reports#monthly_metrics', as: 'monthly_metrics'
```

Comment:
I agree this is something we can probably clean up easily. I don't think it's a problem as long as the links from `/admin/stats` work properly. I'll add a commit to specify this path using a wildcard route based on the controller action, and we can move over the rest in a future commit.